tests/unit/test_project_by_external_id.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+import pytest
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+_project_json = """[{
+  "id": "unify://unified-data/v1/projects/1",
+  "externalId": "project 1 external ID",
+  "name": "project 1 name",
+  "description": "project 1 description",
+  "type": "DEDUP",
+  "unifiedDatasetName": "project 1 unified dataset",
+  "created": {
+    "username": "admin",
+    "time": "2018-09-10T16:06:20.636Z",
+    "version": "project 1 created version"
+  },
+  "lastModified": {
+    "username": "admin",
+    "time": "2018-09-10T16:06:20.851Z",
+    "version": "project 1 modified version"
+  },
+  "relativeId": "projects/1"
+}]
+"""
+
+_project_external_id = "project 1 external ID"
```

Comment:
it's more pythonic to eliminate the leading `_` here since it's generally well understood that you shouldn't be importing/reusing stuff from test files.

Reply:
🤷‍♂️ ok